Update Github workflows to use latest Ubuntu [MA-161]

zachgarwood opened the pull request.

zachgarwood committed:
Update Github workflows to use latest Ubuntu [MA-161]

nikhiltri reviewed (approved):
Looks good! ⭐ But some of the GitHub Actions are failing for what seems to be different reasons. Next time you're on mobile take a look before merging in.

zachgarwood committed:
Update upload artifact action [MA-161]

zachgarwood committed:
Update db credentials in testing workflow [MA-161]

zachgarwood committed:
Removing browser tests [MA-161]

I'm removing the browser test workflow for three reasons:
1. Github Actions is unable to run the workflow
2. I'm also unable to get the browser tests running locally
3. Laravel 12 recommends using Pest 4 for browser tests instead of Dusk

I definitely think we should revisit browser tests in the future, but I
don't think right now is the approprate time to address them.
